src/main/groovy/com/marklogic/gradle/task/roxy/RoxyGenerateTestScaffoldTask.groovy

```diff
@@ -0,0 +1,100 @@
+package com.marklogic.gradle.task.roxy
+
+import org.gradle.api.tasks.TaskAction
+
+//import com.marklogic.gradle.task.roxy.scaffoldTemplates.SampleTestFiles
+
+// Task mlGenerateRoxyTest : This task uses Groovy's SimpleTemplateEngine to customize the sample test suite
+// Parameters:
+//    roxySuitesDir   (Default: roxy/src/test/suites)
+//        - Specify the target directory for the sample test suite
+//    suiteName       (Default: SampleTestSuite)
+//        - Specify the name of the generated test suite
+//    testName        (Default: SampleTest)
+//        - Specify the name of the generated test
+// Examples:
+//    gradle mlGenerateRoxyTest
+//    gradle mlGenerateRoxyTest -PsuiteName="foo" -PtestName="bar" -ProxySuitesDir="./src/roxyTest/suites"
+class RoxyGenerateTestScaffoldTask extends RoxyTask {
+    CommandLineArguments arguments
+
+    @TaskAction
+    void generateScaffoldTask() {
+        println "Generate Roxy Test Scaffold"
```

Comment:
I think this line can be removed, because ml-gradle will first log the name of the task, which pretty much says the same thing - e.g. "mlRoxyGenerateTestSuite". 

But I do think some logging at the end would be helpful - each generated file should be logged. That makes it easy for the end user to know what happened - they can see every file that was created.